### ISSUE TITLE
How to use with Gitlab

### ISSUE THREAD

**Issue by majkinetor** [AlloyCI/alloy_ci#44]
Is there any info on how to setup Gitlab with AlloyCI ?

I find Gitlab CI in some aspects lacking and would like to try it instead of its CI/CD features. 

I could replicate all Github requirements using Gitlab interface, but before doing that I wanted to check out if there there are some docs available already or if there are known problems.

Thanks

**Comment by supernova32** [comment]
Not really. AlloyCI is not compatible with GitLab, nor does it need/want to be. We use the gitlab-runner as an executor, so basically, AlloyCI supports a subset of the features that GitLab CI does. We are inspired by that project, and the aim for 1.0 is to have, at least, the same features.

This means that if you find GitLab CI lacking, AlloyCI will just disappoint you even further :trollface: 

Now, if you'd like to contribute the changes needed for AlloyCI to also be compatible with GitLab, you are more than welcome to do so. That is, however, not a feature the *we ourselves* plan to implement in the future.
